src/pkg/packager/actions.go

```diff
@@ -160,9 +160,12 @@ retryCmd:
 
 	select {
 	case <-timeout:
-		// If we reached this point, the timeout was reached.
-		return fmt.Errorf("command \"%s\" timed out after %d seconds", cmdEscaped, cfg.MaxTotalSeconds)
-
+		// If we reached this point, the timeout was reached or command failed with no retries.
+		if cfg.MaxTotalSeconds < 1 && cfg.MaxRetries < 1 {
+			return fmt.Errorf("command \"%s\" failed (no retries)", cmdEscaped)
+		} else {
+			return fmt.Errorf("command \"%s\" timed out after %d seconds", cmdEscaped, cfg.MaxTotalSeconds)
```

Comment:
```suggestion
			return fmt.Errorf("command %q timed out after %d seconds", cmdEscaped, cfg.MaxTotalSeconds)
```
All of these could be %q to simplify them